src/components/organisms/ChatSidebar/components/PrivateChats/PrivateChats.scss

```diff
@@ -7,40 +7,19 @@
   width: 100%;
 
   &__search {
-    margin: 15px;
-    height: 32px;
-    position: relative;
-  }
-
-  &__search-input {
-    text-align: left;
-    font-size: 14px;
-    background-color: $black;
-    padding-left: 40px;
-    height: 100%;
-    width: 100%;
-
-    &::placeholder {
-      color: $white;
-      opacity: 0.2;
+    margin: 0.9375rem;
+    height: 2rem;
+
+    & input {
+      background-color: $black;
+      text-align: left;
+      font-size: 0.875rem;
+
+      &::placeholder {
+        color: $white;
+        opacity: 0.2;
+      }
     }
```

Comment:
I believe we/BEM prefer using class names for styling, rather than directly styling html elements.